content/projects/oop/person/_index.md

```diff
@@ -58,15 +58,24 @@ System.out.println(greeting)
 This should output:
 
 ```
-Hello, my name is Ryan and I am 30 years old. My interests are being a hardarse, agile and ssd hard drives.
+Hello, my name is Ryan and I am a 30 year old male. My interests are being a hardarse, agile and ssd hard drives.
 ```
 
 In OOP this is known as abstraction. We created a simple model of a more complex thing. We only represent the attributes and functionality that we need.
 
 When an object instance is created from a class, the class's constructor function is run to create it. This process of creating an object instance from a class is called instantiation — the object instance is instantiated from the class. `person` is an instance of `Person`
 
+## Instructions for reviewer
```

Comment:
General clean code should become a habit for both the recruits as well as all reviewers. This should not need to be mentioned in every project. Please remove the instructions relating to general clean code guidelines and only leave the project-specific instructions.

Also please move the review instructions to below the Acceptance criteria.

Reply:
@kaleem99 please double-check, it looks like general clean code guidelines are still being included in the project review instructions